src/libappimage_shared/elf.c

```diff
@@ -1,3 +1,7 @@
+/* Support Large File */
+#define _FILE_OFFSET_BITS 64
```

Comment:
Why not declare publicly in CMake for all targets? Then, dependencies etc. will pick up the flags, too.

Reply:
I guess it will not hurt. Done.